### PR TITLE
feat(hello): add automatic hostname generation

### LINKED ISSUE #14: [FEATURE] hello@1.1.0
**Which chart**:
hello@1.1.0

**Is your feature request related to a problem? Please describe.**
- Allow generating hostname automatically

**Describe the solution you'd like**
By adding `cloudmobility.io/generate-hostname: true` annotation to the ingress will allow to automatically generate the hostname

**Additional context:**
Additional context you feel that may be relevant.

### PR DESCRIPTION
**Pull request by tiagomichaelsousa**
**Which chart**:
hello@1.1.0

**What type of PR is this?**
Feature: a new feature

**What this PR does / why we need it:**
- add automatic hostname to hello tailored chart

**Which issue(s) this PR fixes:**

Closes #14 

**Special notes for your reviewer:**

### CODE REVIEW

File: charts/hello/templates/ingress.yaml [hunk 1]
```diff
@@ -17,8 +18,8 @@ metadata:
   {{- end }}
 spec:
   rules:
-    {{- if $isSharedIngressController}}
-    - host: "{{ $.Release.Name }}-{{ $.Release.Namespace }}-agile-dingo.cloud.eu1.cloudmobility.io"
+    {{- if and $isSharedIngressController $isAutomaticHostnameEnabled}}
```
Comment (jpmarques66) [inline charts/hello/templates/ingress.yaml:21]: Do you think we can remove this verification here? I'm guessing that our webhook will handle the logic of checking if the annotation for automaticDNS is set, and then clearing up all hosts that are not the generated one.

File: charts/hello/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Hello
+
+![hello-chart-badge](https://img.shields.io/badge/Hello%20Chart-1.0.0-green?style=flat-square&logo=mercedes)
+---
+
+Hello is a chart that allow cloudmobility to easily proof the concept of applications inside the cloudmobility
+infrastructure. This chart not meant to be use for production systems since it's just a proof of concept.
```
Comment (jpmarques66) [inline charts/hello/README.md:7]:
```suggestion
Hello is a helm chart that deploys a simple web app and can be used to test cloudmobility Applications Catalog. This chart is not meant to be used in production environments.
```

Comment (jpmarques66) [reply charts/hello/README.md:7]: I'd go for something like this, wdyt?

**Review by jpmarques66 — APPROVED** [review]
Left a couple of comments. Other than that, POG 🏈 .

File: charts/hello/templates/ingress.yaml (hunk 1 above)
Comment (tiagomichaelsousa) [reply charts/hello/templates/ingress.yaml:21]: you are totally right, this will simplify a lot 😎

**Pull request merged**